MaxText/checkpointing.py

```diff
@@ -320,12 +321,20 @@ def load_params_from_path(load_parameters_from_path, abstract_unboxed_params):
   assert load_parameters_from_path, "load_parameters_from_path is not defined."
   max_logging.log(f"restoring params from {load_parameters_from_path}")
   ckpt = epath.Path(load_parameters_from_path)
-  ckptr = ocp.PyTreeCheckpointer()
+  # ckptr = ocp.Checkpointer(ocp.PyTreeCheckpointHandler(restore_concurrent_gb=500, save_concurrent_gb=500))
```

Comment:
You might need something like this if you see an error like `ValueError: Requested more bytes than we reserved space for: 109924319232 > 96000000000`. You need to increase the restore limit. 

The sharded ckpt I linked to in our internal docs should resolve the need for this though